src/Backend/Core/Js/backend.js

```diff
@@ -1844,67 +1843,64 @@ jsBackend.locale = {
 jsBackend.messages = {
   timers: [],
 
-  // init, something like a constructor
-  init: function () {
-    // bind close button
-    $(document).on('click', '#messaging .formMessage .iconClose', function (e) {
-      e.preventDefault()
-      jsBackend.messages.hide($(this).parents('.formMessage'))
-    })
-  },
-
-  // hide a message
-  hide: function (element) {
-    // fade out
-    element.removeClass('active').delay(250).hide(1)
-  },
-
   // add a new message into the que
-  add: function (type, content, optionalClass) {
+  add: function (type, content, optionalClass = '', dismissable = false) {
     var uniqueId = 'e' + new Date().getTime().toString()
 
     // switch icon type
     var icon
+    var role = 'status'
+    var live = 'polite'
+    var dismissableClass = ' d-none'
+
     switch (type) {
       case 'danger':
-        icon = 'times'
+        icon = 'far fa-times-circle'
+        role = 'alert'
+        live = 'assertive'
         break
       case 'warning':
-        icon = 'exclamation-triangle'
+        icon = 'fas fa-exclamation-circle'
+        role = 'alert'
+        live = 'assertive'
         break
       case 'success':
-        icon = 'check'
+        icon = 'far fa-check-circle'
         break
       case 'info':
-        icon = 'info'
+        icon = 'fas fa-info-circle'
         break
     }
 
-    var html = '<div role="alert" id="' + uniqueId + '" class="alert-main alert alert-' + type + ' ' + optionalClass + ' alert-dismissible formMessage ' + type + 'Message">' +
-      '<div class="container-fluid">' +
-      '<i class="fas fa-' + icon + '" aria-hidden="true"></i>' + ' ' +
-      content +
-      '<button type="button" class="close" data-dismiss="alert" aria-label="' + utils.string.ucfirst(jsBackend.locale.lbl('Close')) + '">' +
-      '<span aria-hidden="true" class="fas fa-times"></span>' +
-      '</button>' +
-      '</div>' +
+    if (dismissable) {
+      dismissableClass = ''
+    }
+
+    var html = '<div role="' + role + '" aria-live="' + live + '" id="' + uniqueId + '" class="toast toast-' + type + ' ' + optionalClass + '" data-autohide="false">' +
+      '<div class="toast-body">' +
+        '<button type="button" class="close' + dismissableClass + '" data-dismiss="toast" aria-label="' + utils.string.ucfirst(jsBackend.locale.lbl('Close')) + '">' +
+          '<i class="fas fa-times"></i>' +
+        '</button>' +
+        '<i class="toast-icon ' + icon + '" aria-hidden="true"></i>' + ' ' +
+        content +
+        '</div>' +
       '</div>'
 
     // prepend
-    if (optionalClass === undefined || optionalClass !== 'alert-static') {
-      $('#messaging').prepend(html)
+    if (optionalClass === undefined || optionalClass !== 'toast-inline') {
```

Comment:
Toast-inline comes from alert-inline. It has been used since forever i guess in fork. I'm not sure if it need to be documented. There are examples in the code. Also i don't think we have docs for things like this. I guess if you think we need that, you better make an issue, because we have to add it for a lot of custom classes we have in fork.

Reply:
I think you are reading it wrong, if `optionalClass` is not defined and not (!==) 'toast-inline', the alert/toast will be shown on top floating, otherwise (when its a toast inline) it will be placed inline in the content section

Reply:
ok